<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="G10GGNLT052" name="CompleteFlowTest-Building-NoneV3" tests="21" failures="15" timestamp="10 Sep 2019 04:22:20 GMT" time="138.223" errors="0">
  <testcase name="LoginAPI" time="8.497" classname="com.arcapi.Common.LoginAPITest"/>
  <testcase name="CreateAssetPOSTAPI" time="7.895" classname="com.arcapi.Buildingtestcases.CreateAssetBuildingV3APITest"/>
  <testcase name="SignAgreementRegistrationAPI" time="2.821" classname="com.arcapi.Common.SignAgreementRegistrationAPITest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Common.SignAgreementRegistrationAPITest.SignAgreementRegistrationAPI(SignAgreementRegistrationAPITest.java:32)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- SignAgreementRegistrationAPI -->
  <testcase name="PaymentOrderSubmitPOSTAPI" time="2.262" classname="com.arcapi.Common.PaymentOrderSubmitPOSTAPITest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Common.PaymentOrderSubmitPOSTAPITest.PaymentOrderSubmitPOSTAPI(PaymentOrderSubmitPOSTAPITest.java:33)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- PaymentOrderSubmitPOSTAPI -->
  <testcase name="AddAssestTimeline" time="4.999" classname="com.arcapi.Common.AddAssestTimelinePOSTAPITest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Common.AddAssestTimelinePOSTAPITest.AddAssestTimeline(AddAssestTimelinePOSTAPITest.java:26)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- AddAssestTimeline -->
  <testcase name="AssetDetailAPI" time="2.063" classname="com.arcapi.Buildingtestcases.AssetDetailEmissionFactorTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Buildingtestcases.AssetDetailEmissionFactorTest.AssetDetailAPI(AssetDetailEmissionFactorTest.java:26)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- AssetDetailAPI -->
  <testcase name="MeterCreatePOSTAPI" time="1.861" classname="com.arcapi.Buildingtestcases.MeterCreatePOSTEmissionTest"/>
  <testcase name="MeterBulkConsumptionPOSTAPI" time="2.207" classname="com.arcapi.Buildingtestcases.EnergyMeterBulkConsumptionPOSTEmissionTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Buildingtestcases.EnergyMeterBulkConsumptionPOSTEmissionTest.MeterBulkConsumptionPOSTAPI(EnergyMeterBulkConsumptionPOSTEmissionTest.java:49)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- MeterBulkConsumptionPOSTAPI -->
  <testcase name="GhgGetTestAPI" time="2.078" classname="com.arcapi.Buildingtestcases.GhgMeanEmissionTest"/>
  <testcase name="AssetUpdateforNewAssetPUTAPI" time="2.063" classname="com.arcapi.Buildingtestcases.AssetUpdateBuildingEmissionTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Buildingtestcases.AssetUpdateBuildingEmissionTest.AssetUpdateforNewAssetPUTAPI(AssetUpdateBuildingEmissionTest.java:25)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- AssetUpdateforNewAssetPUTAPI -->
  <testcase name="MetersAPI" time="1.92" classname="com.arcapi.Buildingtestcases.MetersListEmissionTest"/>
  <testcase name="ConsumptionCreatePOSTAPI" time="2.181" classname="com.arcapi.Buildingtestcases.ConsumptionCreatePOSTEmissionTest">
    <failure type="java.lang.AssertionError" message="expected [201] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Buildingtestcases.ConsumptionCreatePOSTEmissionTest.ConsumptionCreatePOSTAPI(ConsumptionCreatePOSTEmissionTest.java:34)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ConsumptionCreatePOSTAPI -->
  <testcase name="RecomputePerformanceScoreGet" time="1.994" classname="com.arcapi.Buildingtestcases.RecomputePerformanceScoreBeforeTest">
    <failure type="java.lang.IllegalArgumentException" message="Cannot get property &amp;apos;energy&amp;apos; on null object">
      <![CDATA[java.lang.IllegalArgumentException: Cannot get property 'energy' on null object
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:255)
	at com.jayway.restassured.internal.path.json.JSONAssertion.getAsJsonObject(JSONAssertion.groovy:53)
	at com.jayway.restassured.internal.path.json.JSONAssertion$getAsJsonObject.callCurrent(Unknown Source)
	at com.jayway.restassured.internal.path.json.JSONAssertion.getResult(JSONAssertion.groovy:28)
	at com.jayway.restassured.path.json.JsonPath.get(JsonPath.java:202)
	at com.jayway.restassured.path.json.JsonPath$get$0.call(Unknown Source)
	at com.jayway.restassured.internal.RestAssuredResponseOptionsGroovyImpl.path(RestAssuredResponseOptionsGroovyImpl.groovy:392)
	at com.jayway.restassured.internal.RestAssuredResponseOptionsImpl.path(RestAssuredResponseOptionsImpl.java:195)
	at com.arcapi.Buildingtestcases.RecomputePerformanceScoreBeforeTest.RecomputePerformanceScoreGet(RecomputePerformanceScoreBeforeTest.java:20)
Caused by: java.lang.NullPointerException: Cannot get property 'energy' on null object
	at org.codehaus.groovy.runtime.NullObject.getProperty(NullObject.java:60)
	at org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:172)
	at org.codehaus.groovy.runtime.callsite.NullCallSite.getProperty(NullCallSite.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:296)
	at Script1.run(Script1.groovy:1)
	at groovy.lang.GroovyShell.evaluate(GroovyShell.java:591)
	at groovy.lang.GroovyShell.evaluate(GroovyShell.java:629)
	at groovy.lang.GroovyShell.evaluate(GroovyShell.java:600)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrap.invoke(PogoMetaMethodSite.java:190)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:71)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at com.jayway.restassured.internal.path.json.JSONAssertion.eval(JSONAssertion.groovy:76)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
	at com.jayway.restassured.internal.path.json.JSONAssertion.getAsJsonObject(JSONAssertion.groovy:46)
	... 32 more
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- RecomputePerformanceScoreGet -->
  <testcase name="GhgGetTestAPI" time="12.303" classname="com.arcapi.Buildingtestcases.GhgMeanEmissionAfterUpdateTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.arcapi.Buildingtestcases.GhgMeanEmissionAfterUpdateTest.GhgGetTestAPI(GhgMeanEmissionAfterUpdateTest.java:32)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- GhgGetTestAPI -->
  <testcase name="AnalysisDataGetAPI" time="22.737" classname="com.arcapi.Buildingtestcases.AnalysisDataEmissionAPITest"/>
  <testcase name="ConsumptionUpdatePUTAPI" time="2.005" classname="com.arcapi.Buildingtestcases.ConsumptionUpdateEmissionAPITest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Buildingtestcases.ConsumptionUpdateEmissionAPITest.ConsumptionUpdatePUTAPI(ConsumptionUpdateEmissionAPITest.java:32)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ConsumptionUpdatePUTAPI -->
  <testcase name="GhgGetTestAPI" time="11.942" classname="com.arcapi.Buildingtestcases.GhgMeanEmissionAfterIncreaseTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Buildingtestcases.GhgMeanEmissionAfterIncreaseTest.GhgGetTestAPI(GhgMeanEmissionAfterIncreaseTest.java:29)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- GhgGetTestAPI -->
  <testcase name="AnalysisDataGetAPI" time="22.847" classname="com.arcapi.Buildingtestcases.AnalysisDataEmissionAfterIncreaseTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Buildingtestcases.AnalysisDataEmissionAfterIncreaseTest.AnalysisDataGetAPI(AnalysisDataEmissionAfterIncreaseTest.java:31)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- AnalysisDataGetAPI -->
  <testcase name="ConsumptionUpdatePUTAPI" time="2.684" classname="com.arcapi.Buildingtestcases.ConsumptionUpdateEmissionDecreaseAPITest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Buildingtestcases.ConsumptionUpdateEmissionDecreaseAPITest.ConsumptionUpdatePUTAPI(ConsumptionUpdateEmissionDecreaseAPITest.java:32)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ConsumptionUpdatePUTAPI -->
  <testcase name="GhgGetTestAPI" time="2.068" classname="com.arcapi.Buildingtestcases.GhgMeanEmissionAfterDecreaseTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Buildingtestcases.GhgMeanEmissionAfterDecreaseTest.GhgGetTestAPI(GhgMeanEmissionAfterDecreaseTest.java:31)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- GhgGetTestAPI -->
  <testcase name="AnalysisDataGetAPI" time="12.008" classname="com.arcapi.Buildingtestcases.AnalysisDataEmissionAfterDecreaseTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.Utill.Controller.Assertion.verifyStatusCode(Assertion.java:20)
	at com.arcapi.Buildingtestcases.AnalysisDataEmissionAfterDecreaseTest.AnalysisDataGetAPI(AnalysisDataEmissionAfterDecreaseTest.java:31)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- AnalysisDataGetAPI -->
</testsuite> <!-- CompleteFlowTest-Building-NoneV3 -->
